### PR TITLE
Fix Bug 1136741, Removes search_suggestions waffle and related code

### PR DESCRIPTION
**Pull request by schalkneethling**
This removes the `search_suggestions` waffle flag and related code.

### CODE REVIEW

**Comment by schalkneethling** [comment]
@jwhitlock I am doing an initial push to see if any tests fail as I cannot get them running locally.

**Comment by schalkneethling** [comment]
Well, all the tests passed 👍

File: kuma/landing/views.py [hunk 1]
```diff
@@ -32,14 +31,11 @@ def home(request):
...
         community_stats = {'contributors': 5453, 'locales': 36}
 
-    groups = FilterGroup.objects.all()
-    serializer = GroupWithFiltersSerializer(groups, many=True)
     default_filters = Filter.objects.default_filters()
 
     context = {
         'updates': updates,
         'stats': community_stats,
-        'command_search_filters': serializer.data,
```
Comment (jwhitlock) [inline kuma/landing/views.py]: ❤️ removed the unneeded context and builder code.

File: kuma/landing/views.py [hunk 2]
```diff
@@ -8,8 +8,7 @@
 from kuma.core.sections import SECTION_USAGE
 from kuma.core.cache import memcache
 from kuma.feeder.models import Bundle
-from kuma.search.models import Filter, FilterGroup
-from kuma.search.serializers import GroupWithFiltersSerializer
```
Comment (jwhitlock) [inline kuma/landing/views.py]: The ``GroupWithFiltersSerializer`` should also be deleted from ``kuma/search/serializers.py``

**Comment by safwanrahman** [comment]
Can you add the bug number in pull request?

**Comment by schalkneethling** [comment]
> Can you add the bug number in pull request?

@safwanrahman Currently no bug number for these. I should probably log bugs for it though. Will do so for the follow on PRs

**Comment by schalkneethling** [comment]
@jwhitlock updated - Thanks for the review.

**Comment by safwanrahman** [comment]
@schalkneethling Can you please file a bug and add the bug id to the PR? so it will be future reference that why it has been removed.

**Comment by jwhitlock** [comment]
You can use [bug 1136741](https://bugzilla.mozilla.org/show_bug.cgi?id=1136741). The convention is to add it to the front of the commit message, like

```
bug 1136741: Remove search-suggestions flag
```

File: kuma/landing/views.py (hunk 2 above)
Comment (stephaniehobson) [reply kuma/landing/views.py]: I see `FilterWithGroupSerializer` in test_serializers.py as well.

**Comment by stephaniehobson** [comment]
@jwhitlock The original PR shows some changes to the models as well, not sure they're still present though: https://github.com/mozilla/kuma/commit/87a6c400e7075f79f635941d40d57cd35820b8f3

**Comment by stephaniehobson** [comment]
Very through and working flawlessly locally. R+ 🔍 

Leaving in case you want to look at test_serializers.py as part of this PR but hit the merge button tomorrow if not.

**Pull request merged**